sdcm/fill_db_data.py

```diff
@@ -3033,8 +3034,11 @@ def _get_table_name_from_query(query):
     @staticmethod
     def cql_create_simple_tables(session, rows):
         """ Create tables for truncate test """
-        create_query = "CREATE TABLE IF NOT EXISTS truncate_table%d (my_id int PRIMARY KEY, col1 int, value int) " \
-            "with cdc = {'enabled': true, 'ttl': 0}"
+        create_query = "CREATE TABLE IF NOT EXISTS truncate_table%d (my_id int PRIMARY KEY, col1 int, value int) "
+        # Cannot create CDC log for a table,
+        # because keyspace uses tablets.See issue  # 16317.
+        if not is_tablets_feature_enabled(session):
```

Comment:
shouldn't it be backported to branch 6.0?

Reply:
I didn't found this error upon upgrade from 5.4 to 6.0, looks this issue is only relevant only when create table with cdc only for new tables.

Reply:
Still I think we should backport to 6.0

Next minor release should test 6.0.0 -> 6.0.1, and those changes would be needed

Reply:
a suggestion 

we already have `version_cdc_support` which has bunch of relatively old versions

but if one would put the logic to disable cdc related testing into that function.
it would cleanly disable cdc related checks, without doing it in multiple places.

beside this one, that seems to have not be guarded by version_cdc_support

Reply:
@fruch is right, also no one will remember to enable it back.